Pokemon/Pokemon/PokemonTableViewController.m

```diff
@@ -0,0 +1,358 @@
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+     self.navigationItem.rightBarButtonItem = self.editButtonItem;
+    
+    
+    NSArray *fire = @[ @"arcanine", @"charizard", @"charmander", @"charmeleon", @"flareon", @"growlithe", @"magmar", @"moltres", @"ninetales", @"ponyta", @"rapidash", @"vulpix"];
+    NSArray *water = @[ @"blastoise", @"cloyster", @"dewgong", @"goldeen", @"golduck", @"gyarados", @"horsea", @"kingler", @"krabby", @"lapras", @"magicarp", @"poliwag", @"poliwhirl", @"poliwrath", @"psyduck", @"seadra", @"seaking", @"seel", @"shellder", @"slowbro", @"slowpoke", @"squirtle", @"staryu", @"vaporeon", @"wartortle"];
+    NSArray *grass = @[ @"bellsprout", @"bulbasaur", @"exeggcute", @"exeggutor", @"gloom", @"ivysaur", @"oddish", @"tangela", @"venusaur", @"victreebel", @"vileplume", @"weepinbell"];
+    NSArray *psychic = @[ @"abra", @"alakazam", @"drowzee", @"hypno", @"kadabra", @"mew", @"mewtwo", @"mr.mime", @"starmie"];
+    NSArray *rock = @[ @"aerodactyl", @"geodude", @"golem", @"graveler", @"kabuto", @"kabutops", @"omanyte", @"omastar", @"onix"];
+    NSArray *flying = @[ @"pigeot", @"pidgeotto", @"pidgey", @"spearow", @"zapdos", @"zubat"];
+    NSArray *poison = @[ @"arbok", @"beedrill", @"ekans", @"golbat", @"grimer", @"koffing", @"muk", @"nidoking", @"nidoqueen", @"nidoran-f", @"nidoran", @"nidorina", @"nidorino", @"tentacool", @"tentacruel", @"weezing"];
+    NSArray *ice = @[ @"articuno", @"jynx"];
+    NSArray *bug = @[@"beedrill", @"caterpie", @"butterfree", @"kakuna", @"metapod", @"paras", @"parasect", @"pinsir", @"scyther", @"venomoth", @"venonat", @"weedle"];
+    NSArray *normal = @[@"chansey", @"ditto", @"dodrio", @"doduo", @"eevee", @"farfetchd", @"fearow", @"jigglypuff", @"kangaskhan", @"lickitung", @"meowth", @"persian", @"porygon", @"rattatta", @"snorlax", @"tauros"];
+    NSArray *fairy = @[@"clefable", @"wigglytuff"];
+    NSArray *ground = @[@"cubone", @"diglett", @"dugtrio", @"marowak", @"rhydon", @"rhyhorn", @"sandshrew", @"sandslash"];
+    NSArray *dragon = @[@"dragonair", @"dragonite", @"dratini"];
+    NSArray *electric = @[@"electabuzz", @"electrode", @"jolteon", @"magnemite", @"magneton", @"pikachu", @"raichu", @"voltorb"];
+    NSArray *ghost = @[@"gastly", @"gengar", @"haunter"];
+    NSArray *fighting = @[@"hitmonchan", @"machamp", @"machoke", @"machop", @"mankey", @"primeape"];
+    
+    
+    self.tableData = @{@"fire" : fire, @"water" : water, @"grass" : grass, @"psychic" : psychic, @"rock" : rock, @"flying" : flying, @"poison" : poison, @"ice" : ice, @"bug" : bug, @"normal" : normal, @"fairy" : fairy, @"ground" : ground, @"dragon" : dragon, @"electric" : electric, @"ghost" : ghost, @"fighting" : fighting };
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+
+-(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
+{
```

Comment:
Hmm, looks like you're missing part of a method definition here; this brace opens the method, but then it just starts on a new method below without the body of the prepareForSegue method.

You also have some indentation issues in this file. I think this is related; when your code is indented well, it's easier to spot errors like this.